rfb/rfbclient.h

```diff
@@ -370,13 +370,13 @@ typedef struct _rfbClient {
 	LockWriteToTLSProc LockWriteToTLS;
 	UnlockWriteToTLSProc UnlockWriteToTLS;
 
-        /* Hooks for custom rendering
+        /** Hooks for custom rendering
          *
          * VNC rendering boils down to 3 activities:
          * - GotCopyRect: copy an area of the framebuffer
          * - GotFillRect: fill an area of the framebuffer with a solid color
          * - GotBitmap: copy the bitmap in the buffer into the framebuffer
-	 * The client application should either set all three of these or none!
+         * The client application should either set all three of these or none!
```

Comment:
Again, this is a problem that was introduced by the same PR that fixes it. That is not necessary. We can all pretend that no problem was introduced, ever, and that we are geniuses who never introduce problems but only fix them.

In other words: why not squash this change, as well as the previous one, into the first commit?